pkg/lendertesting/zc_token_test.go

```diff
@@ -0,0 +1,65 @@
+package lendertesting
```

Comment:
just like YieldToken -- this should be in `/testing` as its testing a /mock/ specifically